requirements/requirements.txt

```diff
@@ -1,9 +1,7 @@
 torch>=1.8
-torchvision>=0.9
 numpy
 tqdm
 psutil
-tensorboard
```

Comment:
Our hook may use tensorboard. Do we have tips if users use the hook using tensorboar without installing it?

Reply:
We only import tensorboard within the `__init__` of the hook. This is implemented so that tensorboard is not a must. The change in the `requirement.txt` will not break colossalai.